Documentation/ContentObjects/Fluidtemplate/Index.rst

```diff
@@ -105,13 +105,14 @@ extbase.controllerExtensionName
 
     Sets the extension name of the controller.
 
-    ..  tip::
-        Setting this allows you to skip the `extensionName` argument for the
-        :ref:`f:translate <t3viewhelper:typo3-fluid-translate>` and
-        the :ref:`f:uri.resource <t3viewhelper:typo3-fluid-uri-resource>` ViewHelpers.
+    ..  warning::
+        Up to TYPO3 11, this property could be used as an alternative to the `extensionName`
+        argument in the ViewHelpers :ref:`f:translate <t3viewhelper:typo3-fluid-translate>`
+        and :ref:`f:uri.resource <t3viewhelper:typo3-fluid-uri-resource>`, provided that
+        translations and public images were stored in the usual paths in your extension.
 
-        This requires you to put translations and public images in the
-        :ref:`usual paths in your extension <t3coreapi:extension-reserved-folders>`.
+        As of TYPO3 v12 this is no longer supported and it is recommended to use absolute keys
+        or the `extensionName` argument plus key arguments in the ViewHelper.
```

Comment:
```suggestion
        or the `extensionName` argument plus key argument (for example, :html:`key="LLL:EXT:my_extension/Resources/Private/Language/locallang.xlf:myKey"`) in the ViewHelper.
```

Reply:
The example should go after "use absolute keys".